lib/rutema/core/configuration.rb

```diff
@@ -201,8 +201,11 @@ def load_configuration filename
         begin 
           cfg_txt=File.read(filename)
           cwd=File.expand_path(File.dirname(filename))
+          #WORKAROUND for ruby 2.3.1
+          fname=filename.dup
```

Comment:
Now that I read it carefully, we just might get away with doing fname=File.basename(filename) which makes the patch a one liner ;)